easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,422 @@
+name = 'Python'
+version = '2.7.18'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['da3080e3b488f648a3d7a4560ddee895284c3380b11d6de75edb986526b9a814']
+
+dependencies = [
+    ('binutils', '2.37'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.36'),
+    ('GMP', '6.2.1'),  # required for pycrypto
+    ('libffi', '3.4.2'),  # required for cryptography
+    ('OpenSSL', '1.1', '', True),
+]
+
+install_pip = True
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 8 January 2022, to the most recent versions still supporting Python 2.x
+exts_list = [
+    ('setuptools', '44.1.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
+    }),
+    ('wheel', '0.37.1', {
+        'checksums': ['e9a504e793efbca1b8e0e9cb979a249cf4a0a7b5b8c9e8b65a5e39d49529c1c4'],
+    }),
+    ('pip', '20.3.4', {
+        'checksums': ['6773934e5f5fc3eaa8c5a44949b5b924fc122daa0a8aa9f80c835b4ca2a543fc'],
+    }),
+    ('nose', '1.3.7', {
+        'checksums': ['f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98'],
+    }),
+    ('blist', '1.3.6', {
+        'checksums': ['3a12c450b001bdf895b30ae818d4d6d3f1552096b8c995f0fe0c74bef04d1fc3'],
+    }),
+    ('paycheck', '1.0.2', {
+        'checksums': ['6db7fc367c146cd59d2327ad4d2d6b0a24bc1be2d6953bb0773cbf702ee1ed34'],
+    }),
+    ('pbr', '5.8.0', {
+        'checksums': ['672d8ebee84921862110f23fcec2acea191ef58543d34dfe9ef3d9f13c31cddf'],
+    }),
+    ('Cython', '0.29.26', {
+        'checksums': ['af377d543a762867da11fcf6e558f7a4a535ff8693f30cce123fab10c00fa312'],
+    }),
+    ('six', '1.16.0', {
+        'checksums': ['1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926'],
+    }),
+    ('toml', '0.10.2', {
+        'checksums': ['b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f'],
+    }),
+    ('setuptools_scm', '4.1.2', {
```

Comment:
According to pypi the latest to support 2.7 is 5.0.2

Reply:
Yes, but an older `setuptools_scm` is required for one of the extensions below.

Reply:
Not according to the versions I have in my .eb I use 5.0.2 and have no problems

Reply:
Hmm, ok then. I could've sworn there was a reason I stuck to an older version here...